design-collaboration-guidelines.md

```diff
@@ -0,0 +1,82 @@
+# Design Collaboration Guidelines
+
+As an app developer to provide the best version of the design, you need to define a standard vocabulary between the design team and you.
+
+Designers are more and more integrating design systems as part of their work, and we can benefit from that to layout common ground. Here below we would define the minimum standard requirements to keep consistency and the usuals forgotten piece of app design. Independently of the size of your team, there is already a set of best practices applied by your designer that you can benefit, and other you will need anyway at some point during the development of the app.
+
+* [A COMPREHENSIVE GUIDE TO DESIGN SYSTEMS](https://www.invisionapp.com/blog/guide-to-design-systems/)
+
+## Style guide - Foundation
+These are the basic style that will serve as the foundation of all other components. And avoid mistakes by not using the proper element. Independently if you follow material design, fluent design or any other guidelines, these basics style element will facilitate design adjustment and rebranding.
```

Comment:
I'd rewrite ``And avoid mistakes by not using the proper element.``  as ``This document will be helpful when avoiding mistakes by not using the proper element as part of the application UI.``What do you think?